.github/workflows/publish_to_pypi.yaml

```diff
@@ -0,0 +1,61 @@
+name: Publish Python distributions to PyPI and TestPyPI
+
+on: [push, pull_request]
+
+jobs:
+  build-n-publish:
+    name: Build and publish to PyPI and TestPyPI
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        # Adding -l {0} helps ensure conda can be found properly
+        shell: bash -l {0}
+    env:
+      ENV_NAME: hera_opm_publish
+      PYTHON: 3.9
+    steps:
+      - uses: actions/checkout@main
+        with:
+          fetch-depth: 0
+
+      - name: Setup Minimamba
+        uses: conda-incubator/setup-miniconda@v2
+        with:
+          miniforge-variant: Mambaforge
+          miniforge-version: latest
+          use-mamba: true
+          python-version: ${{ env.PYTHON }}
+          environment-file: ci/${{ env.ENV_NAME }}.yml
+          activate-environment: ${{ env.ENV_NAME }}
```

Comment:
In this case, you really don't need to use conda at all, you could just use the `setup-python` action. However, perhaps minimamba is actually faster. I know standard pip is faster than standard conda.

Reply:
Good point. The dependencies are probably lightweight enough that `setup-python` should work and I can use pip to get what I need.